configure.ac

```diff
@@ -463,6 +463,7 @@ AS_IF([test "x$enable_defaultflags" = "xyes"],
       AS_IF([test "x$ax_is_release" = "xno"],
           [ADD_COMPILER_FLAG([-Werror])])
       ADD_COMPILER_FLAG([-fstack-protector-all])
+      ADD_COMPILER_FLAG([-Wno-error=address-of-packed-member])
```

Comment:
IMHO this should not be turned of globally but locally with a #pragma

Reply:
I would ask why? Also pragma's are less portable. Ie if someone has to take the source and add a build system, that's one thing they can carry out of tree easily, but modifying the source to support pragma means carrying an out of tree inter-source file patch.